sigridci/sigridci.py

```diff
@@ -271,23 +269,31 @@ def generate(self, feedback, args):
         print("-" * self.LINE_WIDTH)
         print("")
         for metric in self.REFACTORING_CANDIDATE_METRICS:
-            print("")
-            print(metric.replace("_PROP", "").title().replace("_", " "))
-            for rc in self.getRefactoringCandidates(feedback, metric):
-                print(self.formatRefactoringCandidate(rc))
+            self.printMetric(feedback, metric)
 
         print("")
         print("-" * self.LINE_WIDTH)
         print("Maintainability ratings")
         print("-" * self.LINE_WIDTH)
-        print("System property".ljust(40) + f"Baseline ({self.formatBaselineDate(feedback)})    New code quality")
+        print("System property".ljust(40) + f"Baseline ({self.formatBaselineDate(feedback)})    New/changed code")
         for metric in self.METRICS:
             if metric == "MAINTAINABILITY":
                 print("-" * self.LINE_WIDTH)
-            self.printRatingColor(metric.title().replace("_", " ").ljust(40) + \
+            self.printRatingColor(metric.replace("_PROP", "").title().replace("_", " ").ljust(40) + \
                 "(" + self.formatRating(feedback["overallRatings"], metric) + ")".ljust(21) + \
                 self.formatRating(feedback["newCodeRatings"], metric), feedback["newCodeRatings"].get(metric))
                 
+    def printMetric(self, feedback, metric):
+        print("")
+        print(metric.replace("_PROP", "").title().replace("_", " "))
+        
+        refactoringCandidates = self.getRefactoringCandidates(feedback, metric)
+        if len(refactoringCandidates) == 0:
+            print("    None")
```

Comment:
I assume these extra whitespaces are here for visual formatting only?

Reply:
You mean the extra whitespace in front of the "none" ?

Reply:
yes exactly

Reply:
It's for readability in the ASCII art output, e.g.:

```
Duplication
    None

Unit Size
    None
```